bin/yaml/libraries.yaml

```diff
@@ -529,6 +529,16 @@ libraries:
       - 1.9.4
       - 1.9.5
       type: github
+    kokkos:
+      build_type: cmake
+      extra_cmake_arg:
+      - -DKokkos_ENABLE_SERIAL=ON
```

Comment:
It is not really necessary to explicitly enable the serial backend.

Reply:
Removed.